ln-gateway/src/lib.rs

```diff
@@ -27,6 +27,8 @@ use tokio::sync::{mpsc, oneshot};
 use tracing::{debug, error, instrument, warn};
 use webserver::run_webserver;
 
+pub type LnGatewayResult<T> = std::result::Result<T, LnGatewayError>;
```

Comment:
Let's just call it Result. We do that elsewhere and I think it's understandable.

Reply:
Result is problematic because serde_hex_deserializer also returns a Result, but it has a different error type (D::Error). I tried to see if I could add a new enum type to LnGatewayError to encapsulate the error that is returned from serde_hex_deserializer, but didn't have much luck getting that to work (I can keep trying this if that's the recommended way of handling this).

Reply:
Understood. I would just call it `Result`, and then have the ones on line 259 and 358 be `std::result::Result`. I made that change locally and the compiler seems to accept it.